src/Test.FunctionalTests.Sarif/SarifValidatorTests.cs

```diff
@@ -192,7 +192,7 @@ public void ValidatesAllTestFiles()
             @"v2\ConverterTestData",
             @"v2\SpecExamples",
             @"v2\ObsoleteFormats",
-            @"..\..\Test.UnitTests.Sarif\net461\TestData"
+            @"..\..\Test.UnitTests.Sarif\netcoreapp2.1\TestData"
```

Comment:
>netcoreapp2 [](start = 41, length = 11)

we no longer build test binaries for net461, only for netcoreapp2.1. eliminates some unnecessary redundancy in testing.